lib/Dialect/MSFT/Transforms/MSFTLowerInstances.cpp

```diff
@@ -166,21 +114,6 @@ void LowerInstancesPass::runOnOperation() {
   }
   if (numFailed)
     signalPassFailure();
-
-  // Since applying a large number of attributes is very expensive in MLIR (both
-  // in terms of time and memory), bulk-apply the attributes necessary for
-  // `hw.globalref`s.
-  for (auto opRefPair : globalRefsToApply) {
-    ArrayRef<hw::GlobalRefAttr> refArr = opRefPair.getSecond();
-    SmallVector<Attribute> newGlobalRefs(
-        llvm::map_range(refArr, [](hw::GlobalRefAttr ref) { return ref; }));
-    Operation *op = opRefPair.getFirst();
-    if (auto refArr =
-            op->getAttrOfType<ArrayAttr>(hw::GlobalRefAttr::DialectAttrName))
-      newGlobalRefs.append(refArr.getValue().begin(), refArr.getValue().end());
-    op->setAttr(hw::GlobalRefAttr::DialectAttrName,
-                ArrayAttr::get(ctxt, newGlobalRefs));
-  }
 }
```

Comment:
I'm glad to see that this is removing a bunch of code. 😉

Reply:
Same.